site/reference/annotations/README.md

```diff
@@ -0,0 +1,29 @@
+# Annotations Reference
+
+<!--mdtogo:Short
```

Comment:
It is unclear to me how the strings produced by these directives will be used. For the actual cli commands, we reference them in the cobra commands, but it is not clear where we would reference this content.
I think we can just remove them.

Reply:
I mostly just copied what was in the CLI reference. It's not clear to me what the mdtogo tags are for.

Reply:
removed